docusaurus/docs/themes/transportation/index.mdx

```diff
@@ -2,45 +2,89 @@ import DocCardList from '@theme/DocCardList';
 
 # Transportation
 
-Overture Transportation data includes features and attributes that describe the infrastructure and conventions of how people and objects move about the world.
+Overture's Transportation theme schema describes the infrastructure and
+conventions of how people and objects travel around the world. Transportation
+data includes highways, footways, cycleways, railways, ferry routes, and public
+transportation.
 
-## Key Schema Design Choices
+<!--
+High-level description above 👆 lifted from:
+  https://wiki.overturemaps.org/display/PROJ/Transportation+Task+Force
+-->
 
-### Scoped Properties
-Overture supports rule-based properties to apply properties that only apply to sub-segments, certain directions of travel, points-in-time, environmental conditions, etc. Geometric scoping may be used to represent static value properties that are "always there."
+## Use Cases
 
-### Segmentation and Connector Model
-Overture models the transportation network using Segments that embody sections of road/trail/path that have similar properties and Connectors that indicate where, for given modes of transportation, it is possible to transition from one segment to another.
+Some common use cases for which data in Overture's Transportation theme schema
+may be used include:
 
+- **Mapping.** Rendering a map of connected roads and paths.
+- **Routing.** Calculating optimal routes from places to place.
```

Comment:
```from place to place``` instead of ```places to place```?

Reply:
Thanks! Fixed that in an upcoming update to the PR.